.travis.yml

```diff
@@ -35,6 +35,7 @@ sudo: false
 script:
  - go get -d -t -v ./...
  - go install -v $TAGS ./...
+ - sed -i "s/\(const debugStackDepth = \)false/\1true/" ./exec/vm.go
```

Comment:
couldn't we instead use build-tags?
we could put the `debugStackDepth=true` in a, e.g., `exec/vm_debug_stack.go` file with some build-tag (say `debugstack`) and the one with `=false` in `exec/vm_stack.go` (with `!debugstack`)